src/Servers/Kestrel/tools/CodeGenerator/HttpProtocolFeatureCollection.cs

```diff
@@ -40,7 +40,8 @@ public static string GenerateFile()
             "IHttpUpgradeFeature",
             "IHttpWebSocketFeature",
             "IHttpWebTransportFeature",
-            "IBadRequestExceptionFeature"
+            "IBadRequestExceptionFeature",
+            "IHttpRequestTimeoutFeature"
```

Comment:
Don't you need to re-run the code generator now?

Reply:
Yup, done.